docs/administering/offline.md

```diff
@@ -0,0 +1,100 @@
+This document explains how to run a Sandstorm server that has no access to the internet ("airgapped" or "offline"). Please note that running such a server requires a bit of work, but it can be done.
```

Comment:
Consider summarizing the things you will need, or might want, to do before going into how to do each of them. Something like:

* Have wildcard DNS
* Have a login service
* Install offline and disable updates
* Make a plan to do manual updates of Sandstorm
* Manually install apps OR set up app index